public/docs/js/latest/guide/setup.jade

```diff
@@ -39,41 +28,35 @@
     application.
 
   p.
-    The TypeScript setup includes System.js, a third-party open-source library that adds ES6 module loading functionality to browsers. This step isn't needed for the ES5 version. System requires mapping the code file paths to understand what to be load.
+    The TypeScript setup includes System.js, a third-party open-source library that adds ES6 module loading functionality to browsers. This step isn't needed for the ES5 version.
 
```

Comment:
The text talks about creating a main.ts file, but I suppose technically, the es5 example would be called main.js. I think this probably belongs in a note before the first code-box, something like "Although we work through the examples here in TypeScript, you can also use regular ES5. Click the ES5 link in any code box to see the ES5 JavaScript version. Note that in ES5, you'd want to name your files .js rather than .ts."